packages/xod-deploy/src/toolManager.js

```diff
@@ -0,0 +1,132 @@
+import R from 'ramda';
+import path from 'path';
+import fse from 'fs-extra';
+import fetch from 'node-fetch';
+import tar from 'tar';
+import bz2 from 'unbzip2-stream';
+import { tapP } from 'xod-func-tools';
+
+// =============================================================================
+//
+// Utils
+//
+// =============================================================================
+
+// :: Path -> Boolean
+const isDir = (dir) => {
```

Comment:
are all these moved from `xod-arduino-deploy`?

Reply:
Yep.
I didn't reuse this package, cause `xod-arduino-deploy` should be removed soon.